CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+Changelog
+=========
+
+
+(unreleased)
+------------
+- Initial commit. [Sylvain Hellegouarch]
```

Comment:
That's kind to set my name, but please set yours if you prefer. You're the one who did the work :)

Reply:
Hi Sylvain, I've accepted all your comments, please review my PR "sync-with-sky-and-implement-pr-from-sylvain", thank you.